REVIEW.md

```diff
@@ -0,0 +1,60 @@
+
+Below are concrete examples of how it applies to code review specifically:
+
+### Inclusion
+
+Be welcoming and inclusive. You should proactively ensure that the author is successful. While any particular pull request may not ultimately be merged, overall we want people to have a great experience and be willing to contribute again. Answer the questions they didn't know to ask or offer concrete help when they appear stuck.
+
+### Sustainability
+
+Avoid burnout by enforcing healthy boundaries. Here are some examples of how a reviewer is encouraged to act to take care of themselves:
+
+* Authors should meet baseline expectations when submitting a pull request, such as writing tests.
+* If your availability changes, you can step down from a pull request and have someone else assigned.
+* If interactions with an author are not following the code of conduct, close the PR and raise it up with your Code of Conduct committee or point of contact. It's not your job to coax people into behaving.
+
+### Trust
+
+Be trustworthy. During a review, your actions both build and help maintain the trust that the community has placed in this project. Below are examples of ways that we build trust:
+
+* **Transparency** - If a pull request won't be merged, clearly say why and close it. If a pull request won't be reviewed for a while, let the author know so they can set expectations and understand why it's blocked.
+* **Integrity** - Put the project's best interests ahead of personal relationships or company affiliations when deciding if a change should be merged.
+* **Stability** - Only merge when the change won't negatively impact project stability. It can be tempting to merge a pull request that doesn't meet our quality standards, for example when the review has been delayed, or because we are trying to deliver new features quickly, but regressions can significantly hurt trust in our project.
+
+## Process
+
+1. Do not start reviewing a pull request if it is WIP or is a draft mode until it is marked as **ready to review**.
+2. Do not start reviewing a pull request if automated checks fail. Wait until they are fixed by the author.
+3. All pull requests except those generated by bots MUST have an issue associated. Motivations and designs SHOULD be discussed in the corresponding issue instead of in the pull request.
+4. Do a quick check for the license header for all new files.
+5. When you provide feedback, make it clear if the change must be made for the pull request to be approved, or if it is just a suggestion. Mark suggestions with `nit`.
+6. Reviews from maintainers are required. Only pull requests approved by **at least one repo maintainer** can be merged. If the author is a maintainer, the pull request is required to be approved by other maintainers.
```

Comment:
Maybe more accurate is Reviews from users  listed in CODEOWNERS. For our project, users in CODEOWNERS are all maintainers.

Reply:
Suggest adding the rule: The number of approvals are controlled by the branch policy, and we should follow the `repository -> Settings -> Branches -> branch protection rules`, and the branch protection rules should be agreed and approved by maintainers.